js/modules/k6/crypto/crypto.go

```diff
@@ -151,3 +152,38 @@ func (hasher *Hasher) Digest(outputEncoding string) string {
 
 	return ""
 }
+
+func (c Crypto) CreateHmac(ctx context.Context, algorithm string, key string) *Hasher {
```

Comment:
I think this should be capitalized; `createHMAC()` rather than `createHmac()`. It would be inconsistent with `hmac()`, but I think that'll have to stay for consistency's sake...